### PR TITLE
Move checkpointed image to /var/lib/docker

### PR DESCRIPTION
**Pull request by huikang**
Now the default place is /var/run/docker, which is reset after rebooting the host

Docker-DCO-1.1-Signed-off-by: Hui Kang hkang.sunysb@gmail.com (github: huikang)

### CODE REVIEW

**Comment by huikang** [comment]
refer to the problem described at

https://github.com/xemul/criu/issues/28#issuecomment-140766559

**Comment by boucher** [comment]
The change will have to be confined to docker, and not touch libcontainer.

File: api/server/server_experimental_unix.go [hunk 1]
```diff
@@ -42,6 +42,8 @@ func (s *Server) postContainersCheckpoint(version version.Version, w http.Respon
 		return err
 	}
 
+	criuOpts.ImagesDirectory = s.daemon.Root
```
Comment (boucher) [inline api/server/server_experimental_unix.go]: Will this not just overwrite the option sent on the command line?

Comment (huikang) [inline api/server/server_experimental_unix.go]: @boucher you are right. This needs a fix. Will do soon

**Comment by huikang** [comment]
@boucher I removed the changes to runc. Do you have a minute to review? Thanks.

**Comment by boucher** [comment]
I'm sorry for not looking at this more closely. Would you be willing to rebase it?

**Comment by huikang** [comment]
no problem. I will do it soon.

**Comment by huikang** [comment]
Rebased

**Pull request closed without merging**